src/lib.rs

```diff
@@ -300,6 +300,15 @@ pub type Result<T, E = error::Error> = std::result::Result<T, E>;
 const GITHUB_BASE_URI: &str = "https://api.github.com";
 const GITHUB_BASE_UPLOAD_URI: &str = "https://uploads.github.com";
 
+/// This `include!` gives us pub const _SET_HEADERS_MAP: [(&str, &str)]
+/// generated from Cargo.toml `[package.metadata.github-api].set-headers` array, like
+/// ```
+/// [package.metadata.github-api]
+///
+/// set-headers = ["X-GitHub-Api-Version: 2022-11-28", ]
```

Comment:
Could we call this `headers` instead? TOML is declarative not imperative, so you wouldn't include a verb like "set"